### PR TITLE
gh-460: added 'TimestampSeries' in TimedeltaSeries in series.pyi

### LINKED ISSUE #460: Missing addition for TimedeltaSeries plus TimestampSeries
**Describe the bug**
We pick up TimestampSeries + TimedeltaSeries, but not the reverse.

**To Reproduce**
```python
import pandas as pd

tds = pd.Series(pd.timedelta_range(start="1 day", periods=10))
tss = pd.Series(pd.date_range(start="2012-01-01", periods=10, freq="W-MON"))

plus= tds + tss
plus2 = tss + tds
```
The line with `plus = tds + tss` fails with `pyright`:
```text
tdts.py:6:7 - error: Operator "+" not supported for types "TimedeltaSeries" and "TimestampSeries"
    Operator "+" not supported for types "TimedeltaSeries" and "TimestampSeries" (reportGeneralTypeIssues)
```
But the line with `plus2 = tss + tds` works.

**Please complete the following information:**
 - OS: Windows 10
 - python version:  3.9
 - pyright 1.1.283
 - version of installed `pandas-stubs `1.5.2.221124

**Additional context**
Need to add `TimestampSeries` to the argument list here:
https://github.com/pandas-dev/pandas-stubs/blob/58416f2c8d3dd44f60e0b2a89da1328ae3fbb58b/pandas-stubs/core/series.pyi#L1776

### PR DESCRIPTION
**Pull request by ramvikrams**
- [x] Closes #460 
- [x] Tests added: Please use `assert_type()` to assert the type of any return value

### CODE REVIEW

File: tests/test_timefuncs.py [hunk 1]
```diff
@@ -1054,3 +1054,10 @@ def test_timedelta64_and_arithmatic_operator() -> None:
         assert_type((s1 * td), Never)  # pyright: ignore
         assert_type((s1 / td), Never)  # pyright: ignore
         assert_type((s3 * td), Never)  # pyright: ignore
+
+
+def test_add_and_timestampseries() -> None:
+    tds = pd.Series(pd.timedelta_range(start="1 day", periods=10))
+    tss = pd.Series(pd.date_range(start="2012-01-01", periods=10, freq="W-MON"))
+    plus = tds + tss
+    check(assert_type(plus, "TimestampSeries"), pd.Series)
```
Comment (Dr-Irv) [inline tests/test_timefuncs.py:1063]: can you add `pd.Timestamp` as the last argument to `check`

Also change name to `test_timedeltaseries_add_timestampseries`

**Review by Dr-Irv — CHANGES_REQUESTED** [review]

File: tests/test_timefuncs.py (hunk 1 above)
Comment (ramvikrams) [reply tests/test_timefuncs.py:1063]: Yes i'll do that

**Comment by ramvikrams** [comment]
Yeah I'll look into it

**Comment by ramvikrams** [comment]
Sir I looked into it, it fails because of this ` Failed to install /Users/runner/Library/Caches/pypoetry/artifacts/59/74/28/0da99c4265a039049b4cccc0a54b5103991e1d78c7d85b899291d627c5/black-22.10.0-1fixedarch-cp310-cp310-macosx_11_0_x86_64.whl`, I think it is not able to install but why?

**Comment by Dr-Irv** [comment]
> I think it is not able to install but why?, that is what I am not able to understand

We are not sure.  I Just did a PR and those jobs had no issue, and there is nothing you changed that would cause them to fail.  May have been a momentary hiccup in the CI.

**Comment by ramvikrams** [comment]
The same tests are failing again

**Comment by Dr-Irv** [comment]
Can you try this?  Modify `pyproject.toml` on the line about the `black` version and change it to:
```text
black = ">=22.12.0"
```
then commit and push.

**Comment by ramvikrams** [comment]
> Can you try this? Modify `pyproject.toml` on the line about the `black` version and change it to:
> 
> ```
> black = ">=22.12.0"
> ```
> 
> then commit and push.

yeah

**Comment by ramvikrams** [comment]
All these tests fail but in my local test `poe test` succeds

**Comment by Dr-Irv** [comment]
That was a separate issue that I did a PR for on Friday, and just merged it.  Do `git fetch upstream` then `git merge upstream/main` then push, and I think we'll be good.

**Comment by ramvikrams** [comment]
> That was a separate issue that I did a PR for on Friday, and just merged it. Do `git fetch upstream` then `git merge upstream/main` then push, and I think we'll be good.

yes

**Comment by ramvikrams** [comment]
Done sir, it finishes off nicely

**Review by Dr-Irv — APPROVED** [review]
thanks @ramvikrams

**Comment by ramvikrams** [comment]
> thanks @ramvikrams 

Thanks sir

**Pull request merged**